cmd/pinniped-proxy/src/main.rs

```diff
@@ -26,15 +27,17 @@ mod tls_config;
 
 #[tokio::main]
 async fn main() -> Result<()> {
-    env_logger::Builder::from_default_env().format(|buf, record| {
-        writeln!(
-            buf,
-            "{} [{}] - {}",
-            Local::now().format("%Y-%m-%dT%H:%M:%S%.6f"),
-            record.level(),
-            record.args()
-        )
-    });
+    env_logger::Builder::from_default_env()
+        .format(|buf, record| {
+            writeln!(
+                buf,
+                "{} [{}] - {}",
+                Local::now().format("%Y-%m-%dT%H:%M:%S%.6f"),
+                record.level(),
+                record.args()
+            )
+        })
+        .init();
```

Comment:
Woops - I'd left the `.init()` off when I re-enabled logging in my last PR, but didn't check it (so nothing will be logged without this change).